File: code/bubblesort.java
//BUBBLE SORT
import java.util.*;
class Solution {
  public static void main(String[] args) {
  Scanner scan=new Scanner(System.in);
  int n=scan.nextInt();
  int[] arr=new int[n];
  for(int i=0;i<n;i++){
  arr[i]=scan.nextInt();
  }
  bubblesort(arr);
  for(int i=0;i<n;i++){
  System.out.println(arr[i]);
  }
  }
   public static void bubblesort(int[] nums){
        for(int i=0; i< nums.length-1; i++){
            for(int j=0;j<nums.length-i-1;j++){
                if(nums[j]>nums[j+1]){
                   int temp=nums[j];
                   nums[j]=nums[j+1];
                   nums[j+1]=temp;
                }
            }
        }
    }
}    

File: code/bubble_sort.cpp
/*************************** Bubble sort*****************************/

#include <bits/stdc++.h>
#include "bubble_sort.h"

#define max_size 10

using namespace std;

int main(int argc, char const *argv[])
{
	cout<<endl;
	int arr[max_size],i;
	for (int i = 0; i < max_size; ++i)
	{
		arr[i] = random()%100;
	}

	cout<<"Array before sorting: "<<endl;
	print_array(arr,max_size);						//from common.h
	cout<<endl;
	
	bubble_sort(arr,0,max_size-1);
	
	cout<<"Array after sorting: "<<endl;
	print_array(arr,max_size);

	return 0;
}